expertise/service/expertise.py

```diff
@@ -79,11 +80,19 @@ def _prepare_config(self, request) -> dict:
                         when it is not expected
         """
         # Validate fields
-        config = self._validate_fields(request)
-        self.logger.info(f"Config validation passed - setting server-side fields")
+        validated_request = APIRequest()
+        validated_request.from_json(request)
+        config = JobConfig(self._get_default_config(), validated_request).to_json()
```

Comment:
default_config can be loaded inside of the class, no need to passot as a parameter

Reply:
Hmm I would like to do this but the JobConfig class has no knowledge of the Flask configuration so it would not know where to look to load this data